### DISCUSSION TITLE
Fetch Syslog (CEF2) Format with Token to SO - Integration with Sentinal One

### DISCUSSION THREAD

**Discussion by Ax386** [Security-Onion-Solutions/securityonion#3587] [Unsupported Versions]
Hi,
is there a possibility to fetch CEF2 (or RFC-5424, STIX, IOC)  Syslog Format (with Token) to Security Onion? my source (sentinal one Cloud) understands CEF, CEF2, RFC-5424, STIX, IOC....Formats.. fetch should be initiated by SO...

...fetching logs from azure O365 via wazuh works (python script) works fine... 

any ideas?
thanks a lot for answers, 
Alex

**Comment by weslambert** [comment]
This could be done with a script and Filebeat/file input, but is there no way to push the syslog so you don't have to do it in this manner?

**Reply by Ax386** [reply]
Hi Wes,
thanks for reply.
yes, there is a way to push the syslogs, but my syslog source is cloud based with no vpn possibility between SO and SAAS Service,... so fetching the logs from SO with auth tokens and certificate seams the more secure way (in transit) to get the data into SIEM..
....what do you thinks is the best way, with focus on automatisation in getting the data, and securing data in transit?
thx,
Alex

**Reply by weslambert** [reply]
If you aren't able to push to Security Onion, then it may be best to pull them down with a bash/python script and have Filebeat read them in.  Otherwise, you could try something like the following:
https://www.elastic.co/guide/en/logstash/current/plugins-inputs-http_poller.html (have not tested)
